translate/rus.json

```diff
@@ -188,7 +188,7 @@
 "head color":"цвет головы",
 "head mix":"смесь головы",
 "head size":"размер головы",
-"hide":"скрывать",
+"hide":"скрыть",
```

Comment:
Does the word 'hide' used anywhere than to hide the panels? If so, this change is not even a slight correction but rather shortening of the translation